examples/tube/openfoam3d_abaqus3d/parameters.json

```diff
@@ -9,6 +9,7 @@
       "delta_t": 0.0001,
       "omega": 0.01,
       "save_results": 1,
+      "save_restart": -10,
```

Comment:
-1 instead of -10?

Reply:
This is intended. The meaning is, that every ten time steps, the restart file is replaced. The default (-1) means that it's replaced every time step.
The reason for the change:
Before the OF settings saved the data every time step (writeInterval 1), but also removed the data every two time steps (purgeWrite 2). However, it's not necessary for CoCoNuT that writeInterval stays at 1 and therefore, it makes more sense to simply only write the data every two time steps: writeInterval 2 and purgeWrite 0.
However, if the CoCoNuT setting `save_restart` is left on the default value, it will complain because the data is only written every two time steps, which conflicts with the requirement to be able to restart from te last time step (value -1).
Note that any multiple of -2 would have been possible.

Reply:
Thanks for the clarification!